include/seqan3/core/type_list/all.hpp

```diff
@@ -7,30 +7,13 @@
 
 /*!\file
  * \author Hannes Hauswedell <hannes.hauswedell AT fu-berlin.de>
- * \brief Provides seqan3::type_list and metaprogramming utilities.
+ * \brief [DEPRECATED] Provides seqan3::type_list and metaprogramming utilities.
+ * \deprecated This header will be removed in 3.1. Please use seqan3/utility/type_list/all.hpp instead.
```

Comment:
```suggestion
 * \deprecated This header will be removed in 3.1. Please use seqan3/utility/type_list/all.hpp and 
 *                     seqan3/utility/type_pack/all.hpp instead.
```

Since only type_list traits may not be enough in certain cases?